### DISCUSSION TITLE
Inactive Tabs? How to turn off this terrible feature?

### DISCUSSION THREAD

**Discussion by yoDon** [mozilla-mobile/firefox-ios#10687] [General]
No "feature release"of any product has subtracted more value for me than the introduction of "Inactive Tabs". How do I turn off this abomination?

Like many users, I keep a large number of tabs open. Those tabs are now effectively lost to me. I had dozens of Hacker news posts I would remind myself of that now just show a favicon and "ycombinator.com" Dozens of YouTube videos whose thumbnails would remind me of exercises or experiences or ideas I wanted to think about that now just show a favicon and "YouTube.com" Friends and experiences I wanted to be reminded of that are now just a favicon "Facebook.com"

This is not the place for judgements on whether you keep tabs open or not. This is not the place for lecturing others on "the right way" to use tabs. The fact that you personally have a different process does not invalidate the processes used by a huge number of browser users. Many users do this. We find value in doing so. The fact that you do not does not mean we don't.

This appears to be not just a massive UX failure but an equally massive product management failure. As I look back through the release notes, this "feature" isn't even mentioned. It's the most consequential functionality change I've ever experienced in a shipping app, comparable to the introduction of the ribbon UX in Microsoft Office, and it doesn't even rate mentioning in the minds of the product management team.

also reported as a bug https://github.com/mozilla-mobile/firefox-ios/issues/10686

**Comment by lmarceau** [comment]
Hello! Thanks for opening this discussion. As I see you've found [the answer](https://github.com/mozilla-mobile/firefox-ios/issues/10686#issuecomment-1120455782) on how to disable the feature, I'll close the discussion here just to keep conversation on this feature in one place.
